src/common/utils.ts

```diff
@@ -0,0 +1,70 @@
+import { SceneDataTransformer, SceneObject, SceneObjectState, SceneQueryRunner, VizPanel } from '@grafana/scenes';
+import { lte } from 'semver';
+import { PanelModel } from './dashboard';
+import { config } from '@grafana/runtime';
+
+export async function refreshPanelsInDashboard() {
+  if (isGrafanaLte10()) {
+    refreshAngularDashboard();
+  } else {
+    const panels = await getAllDashboardPanels();
+    panels.forEach((panel) => {
+      const queryRunner = getQueryRunnerFor(panel);
+      queryRunner?.runQueries();
+    });
+  }
+}
+
+async function getAllDashboardPanels(): Promise<VizPanel[]> {
```

Comment:
this could be improved by getting only panels that contain queries with twinmaker datasource, but we can add that later as well